scene/gui/tree.cpp

```diff
@@ -3449,18 +3452,16 @@ void Tree::gui_input(const Ref<InputEvent> &p_event) {
 					pos.y -= _get_title_button_height();
 
 					if (pos.y < 0) {
-						if (mb->get_button_index() == MouseButton::LEFT) {
-							pos.x += cache.offset.x;
-							int len = 0;
-							for (int i = 0; i < columns.size(); i++) {
-								len += get_column_width(i);
-								if (pos.x < len) {
-									cache.click_type = Cache::CLICK_TITLE;
-									cache.click_index = i;
-									//cache.click_id=;
-									update();
-									break;
-								}
+						pos.x += cache.offset.x;
+						int len = 0;
+						for (int i = 0; i < columns.size(); i++) {
+							len += get_column_width(i);
+							if (pos.x < len) {
+								cache.click_type = Cache::CLICK_TITLE;
+								cache.click_index = i;
+								//cache.click_id=;
```

Comment:
```suggestion
```
Don't think a commented out (and unfinished) code is needed.

Reply:
>or will that invalidate your approval if I force push?

Not sure. But even if it would invalidate my approval here I think that anyone would get that I've already approved the actual changes. :slightly_smiling_face:

Reply:
Further updating a PR doesn't invalidate past approvals (at least not automatically - of course if the PR morphs into something completely new a human might ask for re-assessment, but here that's not the case).

Reply:
And so it was pushed.